src/components/NavBar.js

```diff
@@ -57,36 +74,34 @@ class NavBar extends Component {
             <Menu.Item className="header logo" to="/">
               GNOSIS
             </Menu.Item>
-            <Menu.Item as={NavLink} to="/">
-              Home
+            <Menu.Item id="home-button" as={NavLink} to="/">
+            {i18n.t('navbar:home')}
             </Menu.Item>
             <div className="ui simple dropdown item">
-              Categories <i className="dropdown icon" />
+              {i18n.t('navbar:categories')} <i className="dropdown icon" />
               <Menu secondary id="navbar">
                 <Menu.Item
-                  name="environment"
-                  active={activeItem === "home"}
+                  name={i18n.t('navbar:environment')}
                   onClick={this.handleItemClick}
                 />
                 <Menu.Item
-                  name="medicine"
-                  active={activeItem === "home"}
+                  name={i18n.t('navbar:medicine')}
                   onClick={this.handleItemClick}
                 />
                 <Menu.Item
-                  name="outreach"
-                  active={activeItem === "home"}
+                  name={i18n.t('navbar:outreach')}
                   onClick={this.handleItemClick}
                 />
               </Menu>
             </div>
             <Menu.Menu position="right">
               <Menu.Item>
-                <Input icon="search" placeholder="Search..." />
+                <Input icon="search" placeholder={i18n.t('navbar:search')} />
               </Menu.Item>
               {createArticleButton}
               {subscribeButton}
               {loginActions}
+              <UpdateLanguage UpdateLanguage={this.ReRender}/>
```

Comment:
Please use conventional naming for props: `UpdateLanguage ` should be `updateLanguage `

Reply:
Fixed.